devito/operator.py

```diff
@@ -90,7 +90,7 @@ class Operator(object):
     def __init__(self, nt, shape, dtype=np.float32, stencils=[],
                  substitutions=[], spc_border=0, time_order=0,
                  forward=True, compiler=None, profile=False,
-                 cache_blocking=False, block_size=5, cb_inner_dim=False,
+                 cache_blocking=False, block_size=5,
```

Comment:
Can we now set a more realistic default value for `block_size`? 5 seems arbitrary. 

Also, is the `cache_blocking` flag still needed, or should setting `block_size = None` not be enough to disable cache blocking altogether?